Reaktoro/Equilibrium/EquilibriumSolver.cpp

```diff
@@ -111,6 +111,12 @@ struct EquilibriumSolver::Impl
     /// The indices of the inert species (i.e., the species in disequilibrium)
     Indices iis;
 
+    /// The indices of the fluid species (i.e., the species in disequilibrium)
+    Indices ifs;
```

Comment:
Do we need these indices of fluid and solid species in the EquilibriumSolver?

Reply:
Fixed